master/getting-started/kubernetes/installation/hardway/install-cni-plugin.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Install CNI plugin
+canonical_url: 'https://docs.projectcalico.org/v3.8/getting-started/kubernetes/installation/hardway/install-cni-plugin'
+---
+
+Kubernetes uses the Container Networking Interface (CNI) to interact with networking providers like {{site.prodname}}.
+The {{site.prodname}} binary that presents this API to Kubernetes is called the **CNI Plugin** and must be installed
+on every node in the Kubernetes cluster.
+
+## Provision K8s Account for the plug-in
+
+The CNI Plugin interacts with the Kubernetes APIServer while creating pods, both to obtain additional information
+and to update the data store with information about the pod.
+
+On the Kubernetes master node, create a key for the CNI Plugin to authenticate with and certificate signing request.
+
+```
+openssl req -newkey rsa:4096 \
+           -keyout cni.key \
+           -nodes \
+           -out cni.csr \
+           -subj "/CN=calico-cni"
+```
+
+We will sign this certificate using the main Kubernetes CA.
+
+```
+sudo openssl x509 -req -in cni.csr \
+                  -CA /etc/kubernetes/pki/ca.crt \
+                  -CAkey /etc/kubernetes/pki/ca.key \
+                  -CAcreateserial \
+                  -out cni.crt \
+                  -days 365
+sudo chown ubuntu:ubuntu cni.crt
+```
+
+Next, we create a kubeconfig file for the CNI.
+
+```
+APISERVER=$(kubectl config view -o jsonpath='{.clusters[0].cluster.server}')
+kubectl config set-cluster kubernetes \
+    --certificate-authority=/etc/kubernetes/pki/ca.crt \
+    --embed-certs=true \
+    --server=$APISERVER \
+    --kubeconfig=cni.kubeconfig
+
+kubectl config set-credentials calico-cni \
+    --client-certificate=cni.crt \
+    --client-key=cni.key \
+    --embed-certs=true \
+    --kubeconfig=cni.kubeconfig
+
+kubectl config set-context default \
+    --cluster=kubernetes \
+    --user=calico-cni \
+    --kubeconfig=cni.kubeconfig
+
+kubectl config use-context default --kubeconfig=cni.kubeconfig
+```
+
+Copy this `cni.kubeconfig` file to every node in the cluster.
+
+## Provision RBAC
+
+Define a ClusterRole the CNI Plugin will use to access Kubernetes.
+
+```
+kubectl apply -f - <<EOF 
```

Comment:
I guess we can't pull this from our manifest generation / templates the way we currently do it (since CNI is installed alongside node and they share a service account), but I worry this might get out of date quickly otherwise.

Maybe one for a follow on PR.